mainnet/shared/chainspec.json

```diff
@@ -0,0 +1,889 @@
+{
+  "name": "LUKSO Mainnet",
+  "dataDir": "lukso",
+  "engine": {
+    "Ethash": {
+      "params": {
+        "minimumDifficulty": "0x0",
+        "difficultyBoundDivisor": "0x0",
+        "durationLimit": "0xd"
+      }
+    }
+  },
+  "params": {
+    "gasLimitBoundDivisor": "0x400",
+    "accountStartNonce": "0x0",
+    "maximumExtraDataSize": "0x20",
+    "minGasLimit": "0x1388",
+    "networkID": "0x2A",
+    "forkBlock": "0x0",
+    "maxCodeSize": "0x6000",
+    "maxCodeSizeTransition": "0x0",
+    "eip150Transition": "0x0",
+    "eip160Transition": "0x0",
+    "eip161abcTransition": "0x0",
+    "eip161dTransition": "0x0",
+    "eip155Transition": "0x0",
+    "eip140Transition": "0x0",
+    "eip211Transition": "0x0",
+    "eip214Transition": "0x0",
+    "eip658Transition": "0x0",
+    "eip145Transition": "0x0",
+    "eip1014Transition": "0x0",
+    "eip1052Transition": "0x0",
+    "eip1283Transition": "0x0",
+    "eip1283DisableTransition": "0x0",
+    "eip152Transition": "0x0",
+    "eip1108Transition": "0x0",
+    "eip1344Transition": "0x0",
+    "eip1884Transition": "0x0",
+    "eip2028Transition": "0x0",
+    "eip2200Transition": "0x0",
+    "eip2565Transition": "0x0",
+    "eip2929Transition": "0x0",
+    "eip2930Transition": "0x0",
+    "eip1559Transition": "0x0",
+    "eip3198Transition": "0x0",
+    "eip3529Transition": "0x0",
+    "eip3541Transition": "0x0",
+    "terminalTotalDifficulty": "0x0",
```

Comment:
Please verify if there is `terminalTotalDifficultyPassed` flag available in Nethermind

@Wolmin Please consider moving to `/mainnet/nethermind/chainspec.json`

Reply:
I cannot see the passed option, so I assume having a null value replaces that